ui/settings.js

```diff
@@ -127,6 +127,21 @@ function create_settings_ui() {
     $(str).appendTo($('.panel-body'));
 }
 
+function get_icon_map() {
+    fs = require('fs');
+    let icon_dir = 'node_modules/cryptocurrency-icons/svg/color/';
+    let files = fs.readdirSync(icon_dir)
+        .map(function(v) { 
+            return {
+                symbol: v.substr(0,v.indexOf('.')),
+                path: icon_dir + v
+            }; 
+        });
+    return files;
+}
+
+let icon_map_list = get_icon_map();
```

Comment:
don't have this outside here. I would suggest moving it inside [this](https://github.com/rotkehlchenio/rotkehlchen/pull/150/files#diff-052744f5c4c5a56390468a5383ba532bR146) if condition to make sure it's not evaluated multiple times.